pkgs/jovian-greeter/greeter.py

```diff
@@ -91,8 +92,11 @@ def start_session(self, command: List[str], environment: List[str]):
             logging.debug("Failed to stop Plymouth", exc_info=ex)
 
         # greetd before 0.9.0 doesn't support env
```

Comment:
I'm not putting myself on the hook for learning python enough *and* greetd to fix this, if it is fixed. Though that would be an appropriate fix, too, maybe?

Reply:
We should just yeet this tbh, even 24.05 (!) is on a new enough greetd to support doing things the normal way.

Reply:
Feel free to handle it.